fe/fe-core/src/main/antlr4/org/apache/doris/nereids/DorisLexer.g4

```diff
@@ -247,6 +247,7 @@ MACRO: 'MACRO';
 MAP: 'MAP';
 MATCHED: 'MATCHED';
 MERGE: 'MERGE';
+MICROSECOND: 'MICROSECOND';
```

Comment:
i think we do not need to add this keyword currently?

Reply:
Remove the MICROSCOND keyword from the DorisLexer.g4 and DorisParser.g4 files. If only microsecond in DorisParser.g4 are removed, then Nereids cannot parse the SQL will appear.

Reply:
yes